Makefile

```diff
@@ -165,6 +165,9 @@ taskmetadata-validator:
 ecr-execution-role-image:
 	$(MAKE) -C misc/ecr $(MFLAGS)
 
+container-health-check-image:
```

Comment:
Can you also make sure that `clean` target for this gets invoked under the `clean:` section in the makefile?